src/pages/oasis.js

```diff
@@ -0,0 +1,197 @@
+  width: 80%;
+  margin: 0 auto;
+  justify-content: center;
+  margin-bottom: 50px;
+
+  @media (max-width: 1000px) {
+    flex-direction: column;
+    align-items: center;
+  }
+`
+
+const Card = styled.div`
+  border: 3px solid #fff;
+  width: ${props => (props.width ? `${props.width}` : "fit-content")};
+  height: fit-content;
+  position: relative;
+  border-color: ${SB_LIGHT_SALMON};
+  padding: 25px;
+  margin: 20px;
+  width: 300px;
+  min-height: 190px;
+
+  @media (max-width: 1000px) {
+    border-color: #fff;
+    border-top-color: ${SB_LIGHT_SALMON};
+    width: initial;
+    height: initial;
+    margin: 0;
+  }
+`
+const CardTitle = styled.span`
+  background: white;
+  color: ${SB_SALMON};
+  font-family: Montserrat;
+  font-size: 18px;
+  left: 0;
+  top: -15px;
+  padding: 0 20px;
+  position: absolute;
+  width: ${props => props.width};
+`
+
+const BodyText = styled.p`
+  background: white;
+  color: ${SB_NAVY};
+  font-family: Montserrat;
+  font-size: 15px;
+`
+const OasisLogo = styled.img`
+  position: absolute;
+  top: 95px;
+  left: -20px;
+  z-index: -1;
+  opacity: 20%;
+
+  @media only screen and (max-width: 1280px) {
+    display: none;
+  }
+`
+
+const OasisPage = () => {
+  return (
+    <Layout page="oasis">
+      <SEO />
+      <Wrapper>
+        <Title>OASIS</Title>
+        <Text>
+          Oasis is the place to go to gain practical experience&nbsp;
+          <SpecialText>building web and mobile applications.</SpecialText>
+          &nbsp;We make the opportunity to work on a software project available
+          to everyone,&nbsp;
+          <SpecialText>regardless of experience level.</SpecialText>
+        </Text>
+      </Wrapper>
+      <SmallTitle>WHY JOIN OASIS?</SmallTitle>
+      <CardContainer>
+        <Card>
+          <CardTitle width={"200px"}>
+            designed for any experience level
+          </CardTitle>
+          <BodyText>
+            Whether it’s your first time developing a web app or you’re an
+            experienced developer, our mentoring resources can help make your
+            project a success.
+          </BodyText>
+        </Card>
+        <Card>
+          <CardTitle width={"260px"}>
+            bridge the gap between classes&nbsp;&amp;&nbsp;co-op
+          </CardTitle>
+          <BodyText>
+            Participating in Oasis will help you learn co-op applicable skills
+            not taught explicitly in coursework.
+          </BodyText>
+        </Card>
+        <Card>
+          <CardTitle width={"180px"}>pursue project ideas together</CardTitle>
+          <BodyText>
+            Oasis gives you the opportunity to meet motivated people, form
+            teams, come up with an idea, and work together to make it a reality.{" "}
+          </BodyText>
+        </Card>
+      </CardContainer>
+      <OasisLogo src={tree} alt="tree" />
+      <OasisBanner>
+        VISIT&nbsp;
+        <StyledLink href="https://www.oasisneu.com/">OASISNEU.COM</StyledLink>
```

Comment:
Not a big deal but could we change this to oasis.sandboxnu.com? We're planning to switch everything over at the end of the semester so it'd be nice an early start on the link here 

```suggestion
        <StyledLink href="https://oasis.sandboxnu.com/">OASIS.SANDBOXNU.COM</StyledLink>
```

Reply:
Actually I think it's fine to leave as is. I realized next semester we'll continue using a subdomain like we do for [showcase.oasisneu.com](https://showcase.oasisneu.com/). It think the url might get unruly if we make it something like showcase.oasis.sandboxnu.com